images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -306,3 +306,21 @@ function Build-GraalVMTable {
         "Environment variables" = $envVariables
     }
 }
+
+function Build-PackageManagementEnvironmentTable {
+    return @(
+        @{
+            "Name" = "CONDA"
+            "Value" = $env:CONDA
+        },
+        @{
+            "Name" = "VCPKG_INSTALLATION_ROOT"
+            "Value" = $env:VCPKG_INSTALLATION_ROOT
+        }
+    ) | ForEach-Object {
+        [PSCustomObject] @{
+            "Name" = $_.Name
+            "Value" = Get-PathWithLink($_.Value)
```

Comment:
These variables are not symlinks so we don't need to use `Get-PathWithLink` here